lib/pf/config/pfqueue.pm

```diff
@@ -0,0 +1,55 @@
+package pf::config::pfqueue;
+
+=head1 NAME
+
+pf::config::pfqueue - Configuration from conf/pfqueue.conf
```

Comment:
Not sure your module template is correct,

It seems this should only be the name and the description should be below.

Now it's reversed